### DISCUSSION TITLE
error with encryption key

### DISCUSSION THREAD

**Discussion by martyd1901** [authelia/authelia#4124] [Q&A] [answered]
I am running authelia on unraid docker and I have been trying to figure out what is going on with this error message I am getting

`level=error msg="Failure running the storage provider startup check: the encryption key is not valid against the schema check value" stack="github.com/authelia/authelia/v4/internal/commands/root.go:196 doStartupChecks\ngithub.com/authelia/authelia/v4/internal/commands/root.go:84  cmdRootRun\ngithub.com/spf13/cobra@v1.5.0/command.go:876                  (*Command).execute\ngithub.com/spf13/cobra@v1.5.0/command.go:990                  (*Command).ExecuteC\ngithub.com/spf13/cobra@v1.5.0/command.go:918                  (*Command).Execute\ngithub.com/authelia/authelia/v4/cmd/authelia/main.go:10       main\nruntime/proc.go:250                                           main\nruntime/asm_amd64.s:1594                                      goexit"`

any ideas?
The encryption key is a random alphanumeric 64 character code and I get this everytime I try to start authelia

**Comment by james-d-elliott** [comment, marked as answer]
This means the key used to initialize the database is not the key that you currently have configured. Your options are:

1. Provide the original key.
2. Wipe the database and start again.

**Reply by martyd1901** [reply]
Thanks I wiped the database as it was a new one anyway.

**Reply by james-d-elliott** [reply]
Make sure to back it up! :)
